Formula/agda.rb

```diff
@@ -7,18 +7,17 @@ class Agda < Formula
   homepage "https://wiki.portal.chalmers.se/agda/"
 
   stable do
-    url "https://hackage.haskell.org/package/Agda-2.5.4.2/Agda-2.5.4.2.tar.gz"
-    sha256 "f645add8a47a35da3f37757204fa9c80aeb0993d628fc2057fa343e92e579b1f"
+    url "https://hackage.haskell.org/package/Agda-2.6.0/Agda-2.6.0.tar.gz"
+    sha256 "bf71bc634a9fe40d717aae76b5b160dfd13a06365615e7822043e5d476c06fb8"
 
     resource "stdlib" do
       url "https://github.com/agda/agda-stdlib.git",
-          :tag      => "v0.17",
-          :revision => "5819a4dd9c965296224944f05b1481805649bdc2"
+          :tag      => "v1.0.1",
+          :revision => "442abf2b3418d4d488381a2f8ca4e99bbf8cfc8e"
     end
   end
 
   bottle do
-    rebuild 1
```

Comment:
Please leave the `bottle do` block untouched, it will be handled automatically upon merging. Thanks!

Reply:
Will merge this way, but in the future @lembacon is right, please do not modify bottle blocks.

Reply:
Thanks, I will remember this for future updates.